### PR TITLE
feat: short descriptions of the terminology sources

### PR DESCRIPTION
**Pull request by sdevalk**
Fix for https://github.com/netwerk-digitaal-erfgoed/network-of-terms/issues/557.

This PR introduces short descriptions for the sources. The descriptions have been taken from the websites of the sources (if available) and then redacted a bit: not all original descriptions are usable for the NoT (i.e. concise and to the point).

### CODE REVIEW

**Comment by sdevalk** [comment]
@ddeboer Could you have a look? Would these descriptions be usable?

File: packages/network-of-terms-catalog/catalog/datasets/cht.jsonld [hunk 1]
```diff
@@ -22,6 +22,12 @@
   "url": [
     "https://data.cultureelerfgoed.nl/term/id/cht/"
   ],
+  "description": [
+    {
+      "@language": "nl",
+      "@value": "Onderwerpen die het cultureel erfgoedveld in Nederland beslaan"
```
[the remark this answers is not in the document]
Comment (sdevalk) [reply packages/network-of-terms-catalog/catalog/datasets/cht.jsonld]: Much better! Fixed.

**Comment by ddeboer** [comment]
We may also want to require schema:description in the SCHACL.

**Comment by sdevalk** [comment]
> We may also want to require schema:description in the SCHACL.

@ddeboer Excellent! Done.

File: packages/network-of-terms-catalog/catalog/datasets/brinkman.jsonld [hunk 2]
```diff
@@ -22,6 +22,12 @@
   "url": [
     "http://data.bibliotheken.nl/id/thes/"
   ],
+  "description": [
+    {
+      "@language": "nl",
+      "@value": "Onderwerpen waarmee de depotcollectie en de catalogus van deze collectie, de Nederlandse Bibliografie online (ORS), zijn ontsloten"
```
[the remark this answers is not in the document]
Comment (EnnoMeijers) [reply packages/network-of-terms-catalog/catalog/datasets/brinkman.jsonld:28]: Misschien zoiets: Onderwerpen die de Koninklijke Bibliotheek aan publicaties toekent?

[the remark this answers is not in the document]
Comment (ddeboer) [reply packages/network-of-terms-catalog/catalog/datasets/brinkman.jsonld:28]: That’s already better. 😄 But what would be the answer to the question: ‘When should I use this dataset?’?

Perhaps: ‘Onderwerpen voor het beschrijven van publicaties’?

**Pull request merged**